bundle/osl/manifests/logic-operator-rhel8.clusterserviceversion.yaml

```diff
@@ -0,0 +1,529 @@
+                    livenessProbe:
+                      httpGet:
+                        path: /healthz
+                        port: 8081
+                      initialDelaySeconds: 15
+                      periodSeconds: 20
+                    name: manager
+                    readinessProbe:
+                      httpGet:
+                        path: /readyz
+                        port: 8081
+                      initialDelaySeconds: 5
+                      periodSeconds: 10
+                    resources:
+                      limits:
+                        cpu: 500m
+                        memory: 128Mi
+                      requests:
+                        cpu: 10m
+                        memory: 64Mi
+                    securityContext:
+                      allowPrivilegeEscalation: false
+                      capabilities:
+                        drop:
+                          - ALL
+                securityContext:
+                  runAsNonRoot: true
+                  seccompProfile:
+                    type: RuntimeDefault
+                serviceAccountName: logic-operator-rhel8-controller-manager
+                terminationGracePeriodSeconds: 10
+      permissions:
+        - rules:
+            - apiGroups:
+                - ""
+              resources:
+                - configmaps
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+                - patch
+                - delete
+            - apiGroups:
+                - coordination.k8s.io
+              resources:
+                - leases
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+                - patch
+                - delete
+            - apiGroups:
+                - ""
+              resources:
+                - events
+              verbs:
+                - create
+                - patch
+          serviceAccountName: logic-operator-rhel8-controller-manager
+    strategy: deployment
+  installModes:
+    - supported: false
+      type: OwnNamespace
+    - supported: false
+      type: SingleNamespace
+    - supported: false
+      type: MultiNamespace
+    - supported: true
+      type: AllNamespaces
+  keywords:
+    - kogito
+    - serverless
+    - serverlessworkflow
+  links:
+    - name: OpenShift Serverless Logic Operator
+      url: https://kiegroup.github.io/kogito-docs/serverlessworkflow/latest/index.html
+  maturity: alpha
+  provider:
+    name: logic-operator-rhel8
+  relatedImages:
+    - image: registry.redhat.io/openshift4/ose-kube-rbac-proxy@sha256:4564ca3dc5bac80d6faddaf94c817fbbc270698a9399d8a21ee1005d85ceda56
+      name: kube-rbac-proxy
+    - image: registry.redhat.io/openshift-serverless-1-tech-preview/logic-operator-rhel8@sha256:aae0198cbd4a9d92130437d3869b6da8854ba3a7c229956e172b621aac3261f3
```

Comment:
@davidesalerno Shouldn't we change the Operator to use the tag `1.29.0` instead of the sha256 as as you did for file `config/manifests/osl/bases/logic-operator-rhel8.clusterserviceversion.yaml` ?